src/style.css

```diff
@@ -0,0 +1,113 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 400;
+  color: rgba(6, 11, 53, 1);
+  --main-blue: rgba(0, 172, 220, 1);
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 16px;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px ;
+  height: 134px;
+  margin: 16px auto 40px;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
```

Comment:
Follow the checklist , please 
<img width="742" alt="image" src="https://user-images.githubusercontent.com/99403198/214554702-736b974c-381e-42b7-901e-02cb4d28137b.png">